boards/pimoroni-tufty2040/src/st7789_parallel.pio

```diff
@@ -0,0 +1,5 @@
+.program st7789_parallel
+.side_set 1
+
+out pins, 32 side 0
```

Comment:
there's only 8 LCD data pins, why specify 32 here?

Reply:
Judging by the datasheet, the shift register is always filled by 32-bits and since we're using DMA to write a byte at a time, we want to clear the whole thing.